### PR TITLE
lesson 3 work

### PR DESCRIPTION
**Pull request by pcstanton**

### CODE REVIEW

File: lesson03/assignment/index.html [hunk 1]
```diff
@@ -3,13 +3,35 @@
...
+	<link rel="stylesheet" href="css/style.css" />
 	<title>Fun with Constructors!</title>
+	<script src= "https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
 	<script src="index.js"></script>
 </head>
 <body>
-	<h1>Assignment 3</h1>
-	<section class="template-hook">
-		Fill this section with the templated components
+
```
Comment (brycebenson) [inline lesson03/assignment/index.html:16]: For this assignment, we were looking to have the whole component come from your JavaScript template. So for here, you'll want just some sort of container like a div with a class that you can populate from your JavaScript.

File: lesson03/assignment/index.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      "email": "mJordan@bulls.com",
+      "linkedInUrl": "mJordas.linkedinprofile.com",
+      "codeLanguages": [
+         "HTML", "CSS", "JavaScript", "Java", "Spring"
+      ]
+    }
+   ]
+
+
+let contacts = linkedin.map(function(el){
```
Comment (brycebenson) [inline lesson03/assignment/index.js:61]: So you'll basically want to move all of your component HTML into your 'con' variable, and then fill out the data with variables just like you're doing.

File: lesson03/assignment/index.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+       </article>`;
+       return con
+    });
+    
+$("#name").append(contacts);
+
+
+
+
+let details = linkedin.map(function(el){
```
Comment (brycebenson) [inline lesson03/assignment/index.js]: You can just have one instance of the map and then append it to your container hook in your HTML.

**Review by brycebenson — CHANGES_REQUESTED** [review]
Please take another look and revise the HTML/JS as noted.

File: lesson03/assignment/index.html (hunk 1 above)
Comment (pcstanton) [reply lesson03/assignment/index.html:16]: oh yes - this was the key concept I was missing - so stupid on me - I was trying to figure out how I was going to get the think to loop.   But changed a couple things and boom worked. ;-)

File: lesson03/assignment/index.js (hunk 2 above)
Comment (pcstanton) [reply lesson03/assignment/index.js:61]: Yes got it now ;-)

File: lesson03/assignment/index.js (hunk 3 above)
Comment (pcstanton) [reply lesson03/assignment/index.js]: Yep got rid the second one.

**Comment by pcstanton** [comment]
Corrected and ready for another grading round - thanks

**Review by brycebenson — APPROVED** [review]
Great work Patrick

**Pull request merged**